ddl/cluster.go

```diff
@@ -156,44 +237,55 @@ func (w *worker) onFlashbackCluster(d *ddlCtx, t *meta.Meta, job *model.Job) (ve
 		return ver, err
 	}
 
-	// stage 1, check and set FlashbackClusterJobID.
+	// Stage 1, check and set FlashbackClusterJobID, and save the PD schedule.
 	if flashbackJobID == 0 {
 		err = kv.RunInNewTxn(w.ctx, w.store, true, func(ctx context.Context, txn kv.Transaction) error {
 			return meta.NewMeta(txn).SetFlashbackClusterJobID(job.ID)
```

Comment:
Why do we need a `kv` key? Could we a variable like `GC`? At present, some `kv` are replaced with variables, so if it is not necessary, introduce this kind of `kv` as little as possible

Reply:
Sounds good, maybe I can change it to global variables in next PR